WcaOnRails/app/controllers/api/v0/api_controller.rb

```diff
@@ -9,9 +10,13 @@ def bad_api_parameter(e)
 
   DEFAULT_API_RESULT_LIMIT = 20
 
+  def current_resource_owner
```

Comment:
`current_resource_owner` is the `User` identified by the access token in the API request. This method I wrote actually originally came from https://github.com/doorkeeper-gem/doorkeeper#authenticated-resource-owner, but then I changed it to stick the `doorkeeper_token` attribute in there. I'm actually going to cherry-pick some of @viroulep's work on this from here: https://github.com/thewca/worldcubeassociation.org/pull/1463/files#r108824295.